src/config/networks/local.ts

```diff
@@ -22,6 +22,7 @@ const local: NetworkConfig = {
     id: ETHEREUM_NETWORK.LOCAL,
     color: '#E8673C',
     label: 'LocalRPC',
+    isTestNet: true,
```

Comment:
we need to add the key to the `src/config/networks/network.d.ts` declaration file

Reply:
```
export type NetworkSettings = {
  // TODO: id now seems to be unnecessary
  id: ETHEREUM_NETWORK,
  backgroundColor: string,
  textColor: string,
  label: string,
  isTestNet: boolean,
  nativeCoin: Token,
}
```
is already there, right?

Reply:
:open_mouth: indeed :grimacing: